### ISSUE TITLE
Link to agency-provided a11y data

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#542]
"In addition to displaying the pa11y scan results, provides a link to agency provided data on another website for domain and agency totals, per discrete metric."

**Comment by gbinal** [comment]
Some questions:  
* Where should these links go?  
* Which agencies have data to link to?

**Comment by rcbaker2** [comment]
Right now - our thought is the link would appear in the domain tab (refer to example wireframe).  Open to alternative suggestions.

Right now, the primary concern is HHS.  While there is the possibility that additinal agencies may may a strong case, I know of no other agency that actually results currently that Pulse could link to.

**Comment by gbinal** [comment]
Gotcha.  So, something like this?  

![screen shot 2016-09-01 at 2 14 26 pm](https://cloud.githubusercontent.com/assets/633088/18178975/71dab61a-704e-11e6-8b7f-88db3aa11e19.png)

**Comment by rcbaker2** [comment]
yes - with the link title of:  "link to agency derived results"

**Comment by gbinal** [comment]
Okay - how about this?  _Note: I took off the words 'link to' since it's already clearly a hyperlink._

![screen shot 2016-09-01 at 2 40 53 pm](https://cloud.githubusercontent.com/assets/633088/18179855/55037c58-7052-11e6-8991-38720940a3d6.png)

**Comment by gbinal** [comment]
@rcbaker2 - what URL would that be going to?

**Comment by rcbaker2** [comment]
I concur with renaming the link to agency derived results.  I will need to reach out to Mark Urban to explain this accommodation and get him to give me a link.

**Comment by gbinal** [comment]
Just to update here - I tried a quick solution to this (inserting an html hyperlink into that data cell) to no avail.  It broke the table.  

@smarina04, @rcbaker2 - can you confirm if this is a hard requirement for 9/30.  If so, I need to know the destination link and what it should be next to.

**Comment by smarina04** [comment]
Yes, this is a hard requirement. I am even ok with it being pushed a week
for this update (so we'd have it available early October) if that's what it
takes. This is one requirement that sets a strong precedent, so we have to
have it to go "live" internally.

**Comment by fureigh** [comment]
Suggesting a small copy edit: change `agency derived results` to `agency-derived results`.

**Comment by gbinal** [comment]
This shouldn't be a problem. I'm going to close this until we have an actionable change that needs to be implemented.
